accepted/future-releases/0546-patterns/feature-specification.md

```diff
@@ -2875,10 +2886,14 @@ To match a pattern `p` against a value `v`:
     7.  Match the tail elements. If `t > 0`, then for `i` from `0` to `t - 1`,
         inclusive:
 
-        1.  Extract the element value `e` by calling `v[l - t + i]`.
+        1.  Let `s` be the subpattern `i` elements after the rest element.
+
+        2.  If `s` is an identifier pattern whose name is `_` then do nothing
+            for this element.
```

Comment:
Probably wise. Even if you can write `[var x, ...]`, I bet someone will write `[var x, ... _]` anyway. No need to make that extra painful.